lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_drb_patches.rb

```diff
@@ -0,0 +1,17 @@
+require "drb/drb"
+require "drb/unix"
+
+# We are only interested in using UNIX Sockets, but the "feature" of looping
+# through all available protocols leads to issues where a TCP socket is
+# tried to be used when a UNIX socket should be used.
+#
+# To prevent this we override the default protocol from TCP to UNIX and
+# prevent the addition of new protocols by overriding add_protocol
+module DRb::DRbProtocol
```

Comment:
Can we use expanded notation?  We've already achieved extreme hack territory, I don't think we need constant problems too.  😉

Reply:
Just setting you up for a future fix and/or bad joke ;)